var/ramble/repos/builtin/applications/quantum-espresso/application.py

```diff
@@ -0,0 +1,99 @@
+
+    software_spec('impi2021p8', spack_spec='intel-oneapi-mpi@2021.8.0', compiler='gcc12')
+
+    software_spec('quantum-espresso', spack_spec='quantum-espresso@7.1', compiler='gcc12')
+
+    required_package('quantum-espresso')
+
+    input_file('AUSURF112', url='https://github.com/QEF/benchmarks/releases/download/bench0.0/AUSURF112.tgz',
+               sha256='2b71c27801f8abbde05f48297b56b3194be0f83be50c50433f44bb886cac5117',
+               description='Input file for AUSURF112 benchmark')
+
+    input_file('CNT10POR8', url='https://github.com/QEF/benchmarks/releases/download/bench0.0/CNT10POR8.tgz',
+               sha256='65e36acd332c1511cfa9cbdab0170a867b3144c98103c5bff3c5621ef795c00a',
+               description='Input file for CNT10POR8 benchmark')
+
+    input_file('GRIR443', url='https://github.com/QEF/benchmarks/releases/download/bench0.0/GRIR443.tgz',
+               sha256='18016da632cde8624faeb07f3de04151a02f3b885aa4cbc111d6b912215403f8',
+               description='Input file for GRIR443 benchmark')
+
+    input_file('GRIR686', url='https://github.com/QEF/benchmarks/releases/download/bench0.0/GRIR686.tgz',
+               sha256='4ade7691f88dd322b41abecbc08cc55437f9d376142e62b615317598abe498d7',
+               description='Input file for GRIR686 benchmark')
+
+    input_file('PSIWAT', url='https://github.com/QEF/benchmarks/releases/download/bench0.0/PSIWAT.tgz',
+               sha256='698fbdac9f307b9ebc77e70e52574d67adb0f21bfd6874559d633471664b77d6',
+               description='Input file for PSIWAT benchmark')
+
+    # Set of small benchmarks to build a workload from.
+    # input_file('pwscf-small', url='https://github.com/QEF/benchmarks/releases/download/bench0.0/pwscf-small-benchmark.tar.gz',
+    #            sha256='4a17dc3f6df865ca8ab2d34eb9285f27af9b61d7532065353768a2e123610ac0',
+    #            description='Input file for pwscf-small benchmark')
+
+    executable('copy_inputs', 'cp ' + os.path.join(Expander.expansion_str('input_path'), '*')
+               + os.path.join(Expander.expansion_str('experiment_run_dir'), '.'),
+               use_mpi=False)
+    executable('execute', 'pw.x {flags} < {input_file}', use_mpi=True)
+
+    workload_names = ['AUSURF112', 'CNT10POR8', 'GRIR443', 'GRIR686', 'PSIWAT']
+    input_files = ['ausurf.in', 'pw.in', 'grir443.in', 'grir686.in', 'psiwat.in']
+    for wl_name, input_file in zip(workload_names, input_files):
+        workload(wl_name, executables=['copy_inputs', 'execute'], inputs=[wl_name])
+        workload_variable('input_path', default=Expander.expansion_str(wl_name),
+                          description='Path to inputs for ' + wl_name + ' workload',
+                          workloads=[wl_name])
+        workload_variable('input_file', default=input_file,
+                          description='Name of input file for ' + wl_name + ' workload',
+                          workloads=[wl_name])
+
+    log_str = Expander.expansion_str('log_file')
```

Comment:
Is this something we should make consistent across the other packages?

Reply:
I made a task to discuss this.

I think the only action for this PR is if we want this syntax or not. But I think if we want this, this PR would not make it consistent across all other files. (I think NAMD might also use this for part of it though)